options.js

```diff
@@ -1,15 +1,21 @@
-const eslint = require('eslint')
-const path = require('path')
-const pkg = require('./package.json')
+import { dirname, join } from 'node:path'
+import { readFileSync } from 'node:fs'
+import { fileURLToPath } from 'node:url'
+import eslint from 'eslint'
 
-module.exports = {
-  bugs: pkg.bugs.url,
+const __dirname = dirname(fileURLToPath(import.meta.url))
+const json = readFileSync('./package.json', 'utf8')
+const pkg = JSON.parse(json)
```

Comment:
In my opinion, that would be clearer if we use the [URL API](https://nodejs.org/dist/latest-v16.x/docs/api/url.html) directly, instead of using this technique to have `__dirname`, that should be a workaround not a long term solution.

You could do it like so :
```js
import { URL } from 'node:url'
import fs from 'node:fs'

const pkgURL = new URL('./package.json', import meta url)
const pkgJSON = await fs.readFileSync(pkgURL, { encoding: 'utf-8' })
const pkg = JSON.Parse(pkgJSON)
```

**Note:** I don't think `pkg` is a great variable name (I know, it's the old name), we could probably name it `packageJSON` or something more clearer.

Reply:
I switched to use new URL
but i guess the name stays for now unless you have a better suggestion, `package` is a reserved word

```
const packageJSON = readFileSync(pkgURL, { encoding: 'utf-8' })
const package = JSON.parse(pkgJSON)
```